### DISCUSSION TITLE
Hit tracking for deprecation errors?

### DISCUSSION THREAD

**Discussion by mildlygeeky** [craftcms/cms#9878] [Ideas]
Throwing it out there - it would be interesting to be able to see how often a deprecation error is being reached. At the moment, we get the date for "Last Occurrence", but it may also be handy to have a "hit" counter. This would allow developers to easily pick out the ones that are being called the most and prioritize them first.

**Comment by LarkAnspach** [comment]
No, give a r./ running error
